homeworks/igor.doroshenko_igordordev/3-async/script.js

```diff
@@ -0,0 +1,61 @@
+const articlesList = document.querySelector('.articles-area');
+const loadingBLock = document.querySelector('.loading-window');
+const url = 'https://jsonplaceholder.typicode.com/posts';
+const sortSelect = document.querySelector('[article-sort-option]');
+const filterInput = document.querySelector('[article-filter-input]');
+const timeDelay = (milliSecond) => new Promise((r) => setTimeout(() => r(), milliSecond));
+
+function sortArticles(articlesArr, sortCondition) {
+    let articlesSorted = articlesArr.slice();
+    if (sortCondition.value === 'sort-a-z') {
+        articlesSorted.sort((a, b) => ((a.title > b.title) ? 1 : -1));
+    } else if (sortCondition.value === 'sort-z-a') {
+        articlesSorted.sort((a, b) => ((a.title > b.title) ? 1 : -1)).reverse();
+    } else if (sortCondition.value === 'sort-default') {
+        articlesSorted = articlesArr;
+    }
+    return articlesSorted;
```

Comment:
Коли можеш використовуй замість `if ... else if`  конструкцію `if + return`. Зазвичай це більш приємно читати і простіге в цьому розбиратися
```suggestion
    if (sortCondition.value === 'sort-a-z') {
        return articlesSorted.sort((a, b) => ((a.title > b.title) ? 1 : -1));
    }
    if (sortCondition.value === 'sort-z-a') {
        return articlesSorted.sort((a, b) => ((a.title > b.title) ? 1 : -1)).reverse();
    }
    return articlesArr;
```